metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/DefaultRegistryInitializer.java

```diff
@@ -24,14 +24,14 @@
 
 import com.google.common.eventbus.EventBus;
 import com.netflix.servo.DefaultMonitorRegistry;
-import com.netflix.spectator.servo.ServoRegistry;
 
 public class DefaultRegistryInitializer implements MetricsInitializer {
   public static final String SERVO_POLLERS = "servo.pollers";
 
   private GlobalRegistry globalRegistry;
 
-  private ServoRegistry registry;
+  @SuppressWarnings("deprecation")
```

Comment:
seems that we did not strong depend servo
no need to keep it? change to another one?

Reply:
I have create a new issue about it. https://issues.apache.org/jira/browse/SCB-1245 and now, do not have enough time about it